CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+**2.1.1 - 10/16/23**
+
+ - Drop support for python 3.12
```

Comment:
We don't actually need to release to do this, and we never announced that we added support for 3.12 in the changelog. I think it's fine not to put this here.